Users/coafile.rst

```diff
@@ -6,7 +6,7 @@ coala configuration file. It is meant to be rather factual. If you wish
 to learn by example, please take a look at :doc:`Tutorial`. It also
 teaches how to change settings inside a coala file to suit your taste.
 
-Naming, Scope and Location
+Naming, Scope, and Location
 --------------------------
```

Comment:
Add one more `-` to avoid warning `/home/travis/build/coala/documentation/Users/coafile.rst:10: WARNING: Title underline too short.` in `TRAVIS`